scipy/spatial/transform/rotation.py

```diff
@@ -657,3 +658,32 @@ def __mul__(self, other):
         if self._single and other._single:
             result = result[0]
         return self.__class__(result, normalized=True)
+
+    def apply(self, points):
+        """Apply this rotation on a set of points.
+
+        Rotates `points` by the rotation(s) represented in the object. In terms
+        of DCMs, this application is the same as `self.as_dcm().dot(points)`.
+        For an object containing `N` rotations applied on `P` points, returns
```

Comment:
Are you sure this semantics is preferable? My initial thinking was that we would use the same semantics as with rotation composition. That is in "single rotation, multiple vectors", "multiple rotations, single vector" or "stack of rotations, stack of vectors of the same size".

In my practice, I only needed this "stack semantics". That is we have a time series of rotations and vectors in one frame and want to transform all of them to another. I'm not sure that this "dot" semantics would mean.

@larsoner, @adbugger  what do you think?

Reply:
+0.75 for the `stack` semantics. Although I don't think `dot` semantics are "wrong", it makes more sense to stick to the convention, and it is backed by practical application.